androidbrowserhelper/src/main/java/com/google/androidbrowserhelper/trusted/LauncherActivity.java

```diff
@@ -134,6 +142,16 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
             return;
         }
 
+        if (mLaunchImmediately) {
+            this.launchTwa();
```

Comment:
nit: You don't need `this` here. (Too much TS?)

Reply:
Yes... too much TS, but TBH, I already had the habit of using `this` in Java even when not necessary.  Removed.